nncf/torch/extensions/__init__.py

```diff
@@ -68,8 +76,30 @@ def get(self, fn_name: str) -> Callable:
         :return: A callable object corresponding to the requested function.
         """
         if self._loaded_namespace is None:
+            time_limit = int(os.environ.get(NNCF_TIME_LIMIT_TO_LOAD_EXTENSION, 60))
+
+            def extension_loader_timeout_handler(signum, frame):
+                # pylint: disable=line-too-long
+                msg = textwrap.dedent(
+                    f"""\
+                    The extension load function failed to execute within {time_limit} seconds.
+                    To resolve this issue, run the following command:
```

Comment:
You forgot to add why the command would help. `... seconds. This may be due to leftover .lock files from the PyTorch C++ extension build process. If this is the case, running the followind command should help: ...`